subhub/log.py

```diff
@@ -130,12 +132,14 @@ def _setup_once():
     )
 
 
-def get_logger(logger_name=None):
+def get_logger(logger_name=None) -> structlog._config.BoundLoggerLazyProxy:
     global IS_CONFIGURED
     if not IS_CONFIGURED:
         IS_CONFIGURED = True
         _setup_once()
     if logger_name is None:
-        logger_name = inspect.currentframe().f_back.f_globals["__name__"]
+        logger_name = inspect.currentframe().f_back.f_globals[  # type: ignore
+            "__name__"
```

Comment:
this is awful|ugly; can we refactor so black doesnt make it look like this train-wreck?